enterprise_access/apps/subsidy_request/tests/test_utils.py

```diff
@@ -26,7 +26,7 @@ def test_get_user_from_request_session(self):
 
         # Create request with session
         request = HttpRequest()
-        middleware = SessionMiddleware()
+        middleware = SessionMiddleware(mock.Mock())
```

Comment:
Please add documentation link for this change.

Reply:
done